deeposlandia/datasets/add_a_dataset.md

```diff
@@ -0,0 +1,80 @@
+% Author: R. Delhome
+% Date: 18/12/11
+
+# How to add a dataset?
+
+Some steps have to be accomplished:
+
+## Prepare the data repository
+
+Following folders must be created and maintained:
+
++ `data/dataset/input/training/images` must contains raw training images
```

Comment:
must contain, without the "s"

Reply:
Fixed, as all the following lines that contain the same mistake.